### PR TITLE
feat: copy existing workflows to deploy to TESTNET

### PR DESCRIPTION
**Pull request by rbadillap**

### CODE REVIEW

**Comment by rbadillap** [comment]
The error is not related to the deployment pipelines, check [the job for more details](https://github.com/sygmaprotocol/sygma-fee-oracle/actions/runs/3644678959/jobs/6154155972)

File: ecs/task_definition_PROD.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+          "name": "DD_SERVICE",
+          "value": "fee-oracle"
+        },
+        {
+          "name": "APP_MODE",
+          "value": "release"
+        },
+        {
+          "name": "WORKING_ENV",
+          "value": "dev"
```
Comment (freddyli7) [inline ecs/task_definition_PROD.json:33]: will this file be used for PROD? or testnet? if for PROD, this value should be `production` and the `HTTP_SERVER_MODE` should be `release`

**Review by freddyli7 — CHANGES_REQUESTED** [review]

**Comment by freddyli7** [comment]
> The error is not related to the deployment pipelines, check [the job for more details](https://github.com/sygmaprotocol/sygma-fee-oracle/actions/runs/3644678959/jobs/6154155972)

yeah don't worry about e2e test failing, it's because solidity repo has changed and those config in fee oracle are outdated

File: ecs/task_definition_PROD.json (hunk 1 above)
Comment (rbadillap) [reply ecs/task_definition_PROD.json:33]: @freddyli7 do you know what are the values for testnet?

File: ecs/task_definition_PROD.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        },
+        {
+          "name": "CONVERSION_RATE_PAIRS",
+          "value": "eth,usdt,eth,matic,matic,usdt,eth,glmr,glmr,usdt"
+        }
+      ],
+      "secrets": [
+        {
+          "name": "IDENTITY_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/IDENTITY_KEY"
```
[the remark this answers is not in the document]
Comment (rbadillap) [reply ecs/task_definition_PROD.json:75]: @freddyli7 yes, please remember I don't have the same context as the previous devops, so if you have context about the new values I can create another secret and reference it to this parameter. Feel free to contact me and provide the values

**Comment by rbadillap** [comment]
@freddyli7 I made and pushed some changes to reference the proper environment, replacing PROD to TESTNET for a better understanding. If you consider some values doesn't match what you are expecting, just feel free to provide me (here or other channel) the proper value.

**Comment by rbadillap** [comment]
@P1sar there's a security issue that blocks me to merge, I can forcefully merge this branch since my admin permissions, but let me know if you want to address this issue before I merge: https://github.com/sygmaprotocol/sygma-fee-oracle/security/dependabot/1

**Comment by freddyli7** [comment]
@rbadillap after renaming from PROD to TESTNET, those values in ECS file make sense now. Regarding the SSM secrets, I saw you will create those values in SSM parameters that presents in 1Pass, which is originally what I thought it should be. So general idea is that STAGE will use stage values, and TESTNET shall use testnet values even thought values are the same.

**Comment by freddyli7** [comment]
@rbadillap the link you provided for security issue is 404, just in case if you were referring the CI issue, you don't need to worry that CI issue, our team will fix it with another PR

File: ecs/task_definition_TESTNET.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+          "name": "POLYGONSCAN_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/POLYGONSCAN_API_KEY"
+        },
+        {
+          "name": "COINMARKETCAP_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/COINMARKETCAP_API_KEY"
+        },
+        {
+          "name": "MOONSCAN_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/MOONSCAN_API_KEY"
```
Comment (freddyli7) [inline ecs/task_definition_TESTNET.json:91]: after you created secrets in SSM for TESTNET, replace those values in `valueFrom` and then I think fee oracle is good to go

**Comment by rbadillap** [comment]
> @rbadillap the link you provided for security issue is 404, just in case if you were referring the CI issue, you don't need to worry that CI issue, our team will fix it with another PR

This is a screenshot of the link I provided:

image

**Pull request merged**